vowpalwabbit/oaa.cc

```diff
@@ -233,11 +233,17 @@ VW::LEARNER::base_learner* oaa_setup(VW::setup_base_i& stack_builder)
   new_options.add(make_option("oaa", data->k).keep().necessary().help("One-against-all multiclass with <k> labels"))
       .add(make_option("oaa_subsample", data->num_subsample)
                .help("subsample this number of negative examples when learning"))
-      .add(make_option("probabilities", probabilities).help("predict probabilites of all classes"))
+      .add(make_option("probabilities", probabilities).help("predict probabilities of all classes"))
       .add(make_option("scores", scores).help("output raw scores per class"));
 
   if (!options.add_parse_and_check_necessary(new_options)) return nullptr;
 
+  if (probabilities)
+  {
+    options.insert("link", "logistic");
+    options.add_and_parse(new_options);
```

Comment:
I do not think this line is necessary. For reference we add options elsewhere and do not use it (https://github.com/VowpalWabbit/vowpal_wabbit/blob/5f13251cbca146bacc7bb6f4e3c390dd6857117a/vowpalwabbit/cb_explore.cc#L365)